app/build.gradle

```diff
@@ -5,24 +5,16 @@ plugins {
 apply plugin: 'com.android.application'
 
 android {
-    compileSdkVersion 25
-    buildToolsVersion "25.0.3"
+    compileSdkVersion 26
+    buildToolsVersion "26.0.1"
 
     defaultConfig {
         applicationId "org.feedhenry.welcome"
         minSdkVersion 16
-        targetSdkVersion 25
+        targetSdkVersion 26
         versionCode 1
         versionName "1.0"
 
-        jackOptions {
-            enabled true
-        }
-    }
-
-    compileOptions {
-        sourceCompatibility JavaVersion.VERSION_1_8
```

Comment:
Yes, seems we need to keep the referenced lines unless we upgrade the android gradle plugin to 3.0.0-alpha1 - at the moment we're still using 2.3.3 (https://github.com/feedhenry-templates/welcome-android-gradle/blob/master/build.gradle#L8) 

So seems like we need to actually keep the referenced lines (& Jack) unless we want to upgrade the Android plugin for Gradle from 2.3.3 to 3.0.0-alpha1 ?